style.css

```diff
@@ -168,7 +168,7 @@ button {
   font-family: inherit;
   background: #fff;
   color: #396df2;
-  border: 1px solid #396df2;
+  border: 1px solid #fff;
   border-radius: 0.5rem;
   padding: 0.75em;
   font-size: 1.05em;
```

Comment:
- [ ] Good job styling the button but there are some issues you need to fix in order to get your project approved. kindly consider applying the button interactions in :hover and :active states, and we must apply those colors shown in the Figma design in those states.
I can see that these interactions have not been applied to your other buttons as well in your project. kindly consider applying these button interactions. I am uploading Figma design picture to get the appropriate colors for different states. kindly consider applying these same colors.
  - Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
![asd](https://user-images.githubusercontent.com/96683843/212989272-ae03cf9c-19d9-45c1-9610-ad5f69e7b9b8.jpg)